app/governance/components/ProposalTable/ProposalTable.module.scss

```diff
@@ -1,6 +1,5 @@
 .tableContainer {
   width: 100% !important;
-  padding: 10px;
 
   .rowTitle {
     display: -webkit-box;
```

Comment:
can I know why this browser-specific tag was added?

Reply:
We had already discussed this earlier. I had done this to handle the Proposal Title overflow without altering the row height . Will try to look for other alternatives

Reply:
Also we do not have an alternative currently for this. However this is supported by all the famous browsers